### ISSUE TITLE
Formatting of cfhttp and cfhttpparam doesn't seem to conform to the provided rules

### ISSUE THREAD

**Issue by murphydan** [jcberquist/commandbox-cfformat#78]
Based on the `binary_operators.padding` rule being set to true, I would have expected different results for the following case. I would expect this...

```
	function index( event, rc, prc ) {
		cfhttp(
			method='GET',
			charset='utf-8',
			url='https://www.google.com/',
			result='result'
		) {
			cfhttpparam(
				name='q',
				type='url',
				value='cfml',
				result='myResult'
			);
		}
	}
```

To be changed to this...
```
	function index( event, rc, prc ) {
		cfhttp(
			method = 'GET',
			charset = 'utf-8',
			url = 'https://www.google.com/',
			result = 'result'
		) {
			cfhttpparam(
				name = 'q',
				type = 'url',
				value = 'cfml',
				result = 'myResult'
			);
		}
	}
```
And if `alignment.consecutive.assignments` = true, all of the equal signs should be lined up with spaces around them. They are lined up with that set to true, but not with spaces around each side.

I can't find any combination of options to get this to work. Thoughts? I've attached my `.cfformat.json` file (renamed to `cfformat.json.txt`) if that helps.

[cfformat.json.txt](https://github.com/jcberquist/commandbox-cfformat/files/4748206/cfformat.json.txt)

**Comment by elpete** [comment]
If I remember right, tags in script is funky with cfformat.  That's
probably what you are running in to.

I know for my own use I usually wrap these in the ignore comments, but
that's not a fix, just a workaround.

**Comment by jcberquist** [comment]
@elpete is right, these are technically tags (in script) not function calls. The "arguments" are really tag attributes and tag attribute values. There aren't any rules governing these currently - they will always be rendered as `attr="val"`.

**Comment by jcberquist** [comment]
@murphydan: So after reviewing what was the case currently I realized that `cftag()`'s in script currently do make use of the function call settings to determine how to render. Given that, I decided to just go ahead and respect the `binary_operators.padding` as a regular function call would. 

I am not sure how I feel about this approach entirely, but I think adding this at least makes things more consistent. It is out in `v0.15.5`.
